src/Application/VisaModel/View/VisaModelView.php

```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types=1);
 
-namespace App\Application\Visa\View;
```

Comment:
:bug: J'ai essayé de créer un modèle une description vide et 2 visas, et j'ai eu cette erreur :thinking: Et maintenant elle revient à chaque fois que j'accède à la page liste Modèles de visas

Je crois qu'il faut mettre la description en `?string` ici ?

![Screenshot 2024-09-17 at 11-56-17 App_Application_VisaModel_View_VisaModelView __construct() Argument #3 ($description) must be of type string null given (500 Internal Server Error)](https://github.com/user-attachments/assets/44848ea5-4430-438d-abad-31a93667591c)

Reply:
Ah yes, dans la vue j'ai pas traité le cas d'une valeure null. C'est corrigé !